sopel/modules/tell.py

```diff
@@ -76,8 +76,18 @@ def setup(self):
         else:
             f.write('')
             f.close()
-    self.memory['tell_lock'] = threading.Lock()
-    self.memory['reminders'] = loadReminders(self.tell_filename, self.memory['tell_lock'])
+    if 'tell_lock' not in self.memory:
+        self.memory['tell_lock'] = threading.Lock()
+    if 'reminders' not in self.memory:
+        self.memory['reminders'] = loadReminders(self.tell_filename, self.memory['tell_lock'])
+
+
+def shutdown(bot):
+    for key in ['tell_lock', 'reminders']:
+        try:
+            del bot.memory[key]
+        except KeyError:
+            pass
```

Comment:
Yep. Reloading this module should purge ALL the things and reload them from the reminders file on disk. 👍